src/core/ext/transport/chttp2/transport/internal.h

```diff
@@ -439,6 +439,9 @@ typedef struct {
   bool seen_error;
   bool exceeded_metadata_size;
 
+  /** the error that resulted in this stream being removed */
+  grpc_error *removal_error;
```

Comment:
I guess there're plans to have this exposed/be used somewhere? (other than the single use introduced in this PR)